theano/gpuarray/__init__.py

```diff
@@ -45,65 +45,68 @@ def init_dev(dev, name=None):
     global pygpu_activated
     if (pygpu.version.major, pygpu.version.minor) < (0, 6):
         raise ValueError("Your installed version of pygpu is too old, please upgrade to 0.6 or later")
+    # This is for the C headers API
+    if pygpu.gpuarray.api_version()[0] < 0:
+        raise ValueError("Your installed libgpuarray is too old, please update")
+    if init_dev.dnn_version is None and dev.startswith('cuda'):
```

Comment:
I'm a bit confused.
How is this supposed to work if there are several devices, some supporting cuDNN and some not?